examples/payment_splitter/lib.rs

```diff
@@ -34,4 +34,189 @@ pub mod my_payment_splitter {
     }
 
     impl PaymentSplitter for Contract {}
+
+    #[cfg(all(test, feature = "e2e-tests"))]
+    pub mod tests {
+        use openbrush::contracts::payment_splitter::paymentsplitter_external::PaymentSplitter;
```

Comment:
i think it's better to use just `use openbrush::contracts::payment_splitter::*` here